### PR TITLE
index file

### PR DESCRIPTION
**Pull request by ghost**
Updated the index file with headings, input box and submit button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,17 @@
 </head>
 
 <body>
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
+    <br/>
```
Comment (tanyapowell) [inline index.html]: No need to use `<br>`. All spacings can be done within CSS

File: index.html [hunk 2]
```diff
@@ -7,7 +7,17 @@
...
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
+    <form action="/action_page.php">
```
Comment (tanyapowell) [inline index.html]: Are we using php? If so, we'll need a server

File: index.html [hunk 3]
```diff
@@ -7,7 +7,17 @@
...
+        Email: <input type="text" name="Email" value="">
+        <input type="submit" value="Submit">
```
Comment (tanyapowell) [inline index.html]: Should be a `<button>`

File: index.html [hunk 4]
```diff
@@ -7,7 +7,17 @@
 </head>
 
 <body>
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
+    <br/>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
+    <form action="/action_page.php">
+        Email: <input type="text" name="Email" value="">
```
Comment (tanyapowell) [inline index.html]: Why is **Email:** outside of the `<input>`?

Can you write something in the `value` for the placeholder?

File: index.html (hunk 3 above)
Comment (tanyapowell) [reply index.html]: `value` should be **Notify Me**

File: index.html [hunk 5]
```diff
@@ -7,7 +7,17 @@
...
+        <input type="submit" value="Submit">
+    </form>
+    <br/>
```
Comment (tanyapowell) [inline index.html]: See above

File: index.html [hunk 6]
```diff
@@ -7,7 +7,17 @@
...
+    <br/>
+    <i>Twitter</i><i>FaceBook</i><i>Instagram</i>
```
Comment (tanyapowell) [inline index.html]: These should be font awesome icons, as per the wireframes.

Also, they need to link to their social media platforms

File: index.html [hunk 7]
```diff
@@ -7,7 +7,17 @@
 </head>
 
 <body>
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
```
Comment (tanyapowell) [inline index.html]: Looks weird saying **number 1**, either have it **no. 1** or **number one**. Something doesn't seem right with this

**Review by tanyapowell — CHANGES_REQUESTED** [review]
Can you add a screengrab of what it looks like please? Ta 😃

File: index.html (hunk 7 above)
Comment (ghost) [reply index.html]: changed to number one

**Comment by tanyapowell** [comment]
@LouisJarrett have you considered accessibility? If not, I'll open up an issue and assign it to you

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: This has now been removed

File: index.html (hunk 2 above)
Comment (ghost) [reply index.html]: No we are not using PHP. I have removed the form tag

File: index.html (hunk 4 above)
Comment (ghost) [reply index.html]: Email has been removed. I have added "Your email address" in the placeholder

File: index.html (hunk 3 above)
Comment (ghost) [reply index.html]: Have changed to button....also changed the value as requested.

File: index.html (hunk 5 above)
Comment (ghost) [reply index.html]: done

File: index.html [hunk 8]
```diff
@@ -1,13 +1,23 @@
 <!DOCTYPE HTML>
 
 <html>
-
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (tanyapowell) [inline index.html]: `font-awesome` is already in the `master` branch and should be inside the `<head>` tag. 
Can you remove this

File: index.html [hunk 9]
```diff
@@ -1,13 +1,23 @@
...
 </head>
 
 <body>
+    <h1>The Black Binder</h1>
+    <h2>Your number one directory for Black Culture</h2>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
 
+    <input type="text" name="Email" value="Your email address...">
+    <input type="button" value="Notify Me">
+<div class="socialMedia">
```
Comment (tanyapowell) [inline index.html]: Can you indent please

File: index.html [hunk 10]
```diff
@@ -1,13 +1,23 @@
...
   <title>The Black Binder</title>
 </head>
 
 <body>
+    <h1>The Black Binder</h1>
+    <h2>Your number one directory for Black Culture</h2>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
 
+    <input type="text" name="Email" value="Your email address...">
+    <input type="button" value="Notify Me">
```
Comment (tanyapowell) [inline index.html]: can you change this from `<input>` to `<button>`?

**Review by tanyapowell — CHANGES_REQUESTED** [review]

File: index.html (hunk 9 above)
Comment (ghost) [reply index.html]: i have indented

File: index.html (hunk 10 above)
Comment (ghost) [reply index.html]: done

File: index.html (hunk 8 above)
Comment (ghost) [reply index.html]: This has been removed

File: index.html [hunk 11]
```diff
@@ -7,7 +7,17 @@
 </head>
 
 <body>
+    <h1>The Black Binder</h1>
+    <h2>Your number one directory for Black Culture</h2>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
```
Comment (tanyapowell) [inline index.html]: blank line?

Comment (ghost) [reply index.html]: removed

Comment (tanyapowell) [reply index.html]: is it?

**Comment by ghost** [comment]
No, I have not considered accessibility

**Review by tanyapowell — APPROVED** [review]

**Pull request merged**